src/syslog.rs

```diff
@@ -51,12 +48,11 @@ syslog = "3"
 The setup is very similar, except with less configuration to start the syslog logger:
 
 ```rust
-extern crate fern;
 # /*
-extern crate syslog;
-# */ extern crate syslog3 as syslog;
+use syslog;
```

Comment:
We shouldn't need to have a `use` statement here (or the comment block surrounding it) since `syslog` will already be in the namespace as an external crate.

Reply:
So, it looks like at a minimum we need this:

```
# use syslog3 as syslog;
# fn setup_logging() ...
```

Otherwise, when you run tests with both the `syslog-3` and `syslog-4` features enabled, you get a bunch of "trait bound ... is not satisfied" errors. Is that the change you were thinking of?

Reply:
Yeah! That looks good to me.

This example in particular is pretending to be in a context where the syslog version 3 crate is just imported as `syslog`, so in the original version I added `extern crate syslog;` since that would be necessary for a user depending on `syslog = "3"` (rather than `syslog3`).

With the 2018-edition example, we need the use statement, but users depending on `syslog = "3"` shouldn't.

Reply:
👍